configs/ote/face-detection/face-detection-0200/template.yaml

```diff
@@ -0,0 +1,52 @@
+# Description.
+model_template_id: Face_Detection_0200
+name: face-detection-0200
```

Comment:
I'd suggest using "face-detection-0200" as an ID, and something like "Face Detection 0200" as a name. Name is something human-readable and nicely formatted, so should not use underscores and dashes w/o a reason.